crates/wysiwyg/src/dom/dom_methods.rs

```diff
@@ -726,20 +761,24 @@ where
         if (cur_handle == *from_handle
             || (from_handle.is_ancestor_of(&cur_handle)
                 && cur_handle.index_in_parent() == 0))
-            && (1..text_node.data().chars().count()).contains(&start_offset)
+            && (1..text_node.data().chars().count() + 1).contains(&start_offset)
```

Comment:
Could this be written as `..=`? I think this would achieve the aim and having struggled with off by one errors in the past, having fewer +/- 1s in the codebase is an improvement in my eyes